streaming_data_types/histogram_hs00.py

```diff
@@ -141,25 +170,57 @@ def serialise_hs00(histogram):
 
     # Build the data
     data_len = reduce(operator.mul, histogram["current_shape"], 1)
-
-    ArrayDouble.ArrayDoubleStartValueVector(builder, data_len)
-    # FlatBuffers builds arrays backwards
-    for x in reversed(histogram["data"].flatten()):
-        builder.PrependFloat64(x)
-    data_vector = builder.EndVector(data_len)
-    ArrayDouble.ArrayDoubleStart(builder)
-    ArrayDouble.ArrayDoubleAddValue(builder, data_vector)
-    data_offset = ArrayDouble.ArrayDoubleEnd(builder)
-
-    errors_offset = None
-    if "errors" in histogram:
+    if isinstance(histogram["data"], numpy.ndarray):
+        flattened_data = histogram["data"].flatten()
+    else:
+        flattened_data = numpy.asarray(histogram["data"]).flatten()
```

Comment:
```suggestion
    flattened_data = numpy.asarray(histogram["data"]).flatten()
```
it is fine to do this even if `histogram["data"]` is already a numpy.ndarray